Documentation/project-docs/standard-platform.md

```diff
@@ -0,0 +1,362 @@
+# .NET Standard Platform
+
+## Why?
+To provide a more concrete guarantee of binary portability to future .NET-capable platforms with an easier-to-understand platform versioning plan.
+
+Today, Portable Class Libraries (PCL) target an intersection of APIs depending on your platform selection when making the project. This gives you a specific surface area that guarantees you work on the chosen platforms. Those combinations are precomputed to give you the right set of surface area. When these portable libraries are packaged into NuGet, they are expressed with a static set of frameworks e.g. **portable-net45+win8**. While this describes the intent that you want to run on .NET Framework 4.5 and Windows 8.0, it is also restrictive, since new platforms can appear in the future that are perfectly capable of running those PCLs but are blocked due to the platforms that were selected when the project was created. In fact, putting the portable dll inside of a folder with a static list of profiles *essentially* makes it platform-specific. It's no different to doing:
+
+```
+MyLibrary/net45/MyLibrary.dll
+MyLibrary/win8/MyLibrary.dll
+```
+
+The biggest difference is that you wouldn't be able to consume it in a Portable Class Library project type.
```

Comment:
Nit: now that we've defined PCL, we can probably just use the acronym rather than writing it out in each place